tests/integration/models/reddit/test_comment.py

```diff
@@ -182,6 +182,21 @@ def test_refresh__removed_comment(self):
         assert ('This comment does not appear to be in the '
                 'comment tree',) == excinfo.value.args
 
+    def test_refresh__with_reply_sort_and_limit(self):
+        with self.recorder.use_cassette('TestComment.test_refresh__with_reply_sort_and_limit'):
+            comment = Comment(self.reddit, 'e4j4830')
+            comment.reply_limit = 4
+            comment.reply_sort = 'new'
+            comment.refresh()
+            replies = comment.replies
+        lastCreated = float('inf')
```

Comment:
```suggestion
        last_created = float('inf')
```

This is incredibly minor, but I *believe* PRAW's convention is to use `snake_case` rather than `camelCase` for variable names. I can't seem to find a source that says that, however.

Reply:
Fixed!